bundler/spec/runtime/with_unbundled_env_spec.rb

```diff
@@ -72,32 +72,22 @@ def run_bundler_script(env, script)
       bundle_exec_ruby bundled_app("source.rb")
       expect(out).to eq original
     end
-  end
-
-  shared_examples_for "an unbundling helper" do
```

Comment:
It's a little confusing in the diff, but what I did here was move all the tests for RUBYOPT, MANPATH, and RUBYLIB out of the shared "unbundling helper" and made them tests for `original_env`. The shared "unbundling helper" is now much smaller and appears beneath them.